src/main/groovy/com/gladed/androidgitversion/AndroidGitVersion.groovy

```diff
@@ -397,6 +406,9 @@ class AndroidGitVersionExtension {
         /** Most recent version seen */
         String lastVersion = 'unknown'
 
+        /** The resulting output from calling git describe */
+        String outputOfGitDescribe = null
```

Comment:
How about `gitDescribeOutput`.

Can this default to `unknown`? Then we don't need to test for null @140.